spec.md

```diff
@@ -725,7 +725,11 @@ Plugins MAY create 3 types of volumes:
 
 - Empty volumes. When plugin supports `CREATE_DELETE_VOLUME` OPTIONAL capability.
 - From an existing snapshot. When plugin supports `CREATE_DELETE_VOLUME` and `CREATE_DELETE_SNAPSHOT` OPTIONAL capabilities.
+  SP SHOULD create a volume that looks like exact copy of the original snapshotted volume at the time the snapshot was taken.
+  If the newly created volume is larger than the original volume and SP supports `EXPAND_VOLUME` node capability, CO SHOULD call `NodeExpandVolume` to make the extra space on the volume available to the workloads. 
```

Comment:
should we say "original volume" or say - `VolumeContentSource` ?

Reply:
I think usage of `volumeContentSource` is correct but with an additional mention that  `snapshot or PVC` as data source, because may be in future we may  have more data sources which dont support `restore` or `clone` operation ?